src/pybind/mgr/dashboard_v2/frontend/src/styles.scss

```diff
@@ -0,0 +1,2 @@
+/* You can add global styles to this file, and also import other style files */
+@import './openattic-theme.scss';
```

Comment:
Not urgent, but it may make sense to rename this file to `dashboard-theme.scss` at some point.